www/index.html

```diff
@@ -82,8 +82,6 @@
                                     <input type="search" id="prefix" placeholder="Search..." class="form-control" />
                                     <button type="button" class="btn btn-outline-secondary" id="searchArticles"><i class="fas fa-search"></i></button>
                                 </span>
-                            </span>
-                            <span class="col-2">
```

Comment:
@ankur12-1610 Please explain why you have deleted the last two columns of the Bootstrap 12-column grid. You have effectively created a line that only uses 10 of the 12 columns.

Reply:
> @ankur12-1610 Please explain why you have deleted the last two columns of the Bootstrap 12-column grid. You have effectively created a line that only uses 10 of the 12 columns.

Oh! sorry I guess it was by mistake I didn't check the files changed, sorry for this one :')

Reply:
> @ankur12-1610 Please explain why you have deleted the last two columns of the Bootstrap 12-column grid. You have effectively created a line that only uses 10 of the 12 columns.

I've amended the changes, please look into it.